src/Compilers/CSharp/Test/Symbol/Symbols/SymbolErrorTests.cs

```diff
@@ -865,7 +867,8 @@ interface i1
     event myDelegate myevent { add {} remove {} }
 }
 ";
-            CreateCompilation(text, parseOptions: TestOptions.Regular7).VerifyDiagnostics(
+            CreateCompilation(text, parseOptions: TestOptions.Regular7,
+                              targetFramework: TargetFramework.NetStandardLatest).VerifyDiagnostics(
```

Comment:
Why did we need to update the framework here?

Reply:
> Why did we need to update the framework here?

To avoid getting "platform doesn't support" errors depending on what framework is used by default.

---
In reply to: [250375910](https://github.com/dotnet/roslyn/pull/32689#discussion_r250375910) [](ancestors = 250375910)